Classes/Composer/ScriptHandler.php

```diff
@@ -151,4 +156,23 @@ private static function calculateMaximumPackageNameLength(array $modules): int
 
         return $maximumLength;
     }
+
+    /**
+     * @param Event $event
```

Comment:
Please can you add a short description of the functionality here?

Reply:
Done.